File: Fibonacci/PL-01/FibonacciSequence.java
package lnu;

import java.util.ArrayList;
import java.util.List;

public class FibonacciSequence {
    private List<Integer> sequence;
    private int steps;

    public FibonacciSequence() {
        sequence = new ArrayList<>();
        steps = 0;
    }

    public List<Integer> getSequence() {
        return sequence;
    }

    public int getSteps() {
        return steps;
    }

    public void setSequence(List<Integer> sequence) {
        this.sequence = sequence;
    }

    public void setSteps(int steps) {
        this.steps = steps;
    }
}

File: Fibonacci/FY-01/FibonacciSequence.cs
namespace Fibonacci

{
    public class FibonacciSequence
    {
        public List<int> Sequence { get; set; }
        public int Steps { get; set; }

        public FibonacciSequence()
        {
            Sequence = new List<int>();
            Steps = 0;
        }
    }
}